config/targets.json

```diff
@@ -203,5 +203,19 @@
         "text": "ed - college scorecard"
       }
     ]
+  },
+  {
+    "name": "calc",
+    "slack_channel": "calc-data-capture",
+    "links": [
+      {
+        "url": "https://calc-dev.apps.cloud.gov/",
+        "text": "CALC (staging)"
+      },
+      {
+        "url": "https://calc.gsa.gov/",
```

Comment:
As noted in #103, perhaps this should not point to the production URL. Let me know if I should remove that entry from `links`.